plugins/pulp_rpm/plugins/migrations/0001_rpm_add_export_distributor.py

```diff
@@ -3,7 +3,8 @@
 # Migrating all old rpm repositories to have export_distributor along with yum_distributor
 import logging
 
-from pulp.server.db.model.repository import RepoDistributor
+from pulp.server.db.connection import get_collection
```

Comment:
Feel free to ignore this comment, but my personal preference is to import modules and use them like module.function() rather than importing function and using it like this, only because it provides more context when reading the code and becomes clearer which namespace the functions come from. It's a personal preference thing though, so do as you please.